packages/aws-cdk-lib/aws-rds/lib/serverless-cluster.ts

```diff
@@ -272,6 +292,23 @@ export interface ServerlessScalingOptions {
    * @default - automatic pause enabled after 5 minutes
    */
   readonly autoPause?: Duration;
+
+  /**
+   * secondsBeforeTimeout specifies the amount of time, in seconds,
+   * that Aurora Serverless v1 tries to find a scaling point to perform seamless scaling before enforcing the timeout action.
+   *
+   * @default - 300 (5 minutes)
+   */
+  readonly secondsBeforeTimeout? : Duration;
```

Comment:
```suggestion
  readonly secondsBeforeTimeout?: Duration;
```

Reply:
I think that this property needs to be changed to just `timeout`. Since we're providing a Duration here, a user could provide minutes if they wanted to. `readonly timeout?: Duration` makes the most sense